src/formatDate.js

```diff
@@ -8,7 +9,41 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separador = fromFormat[3];
+  const separadorTo = toFormat[3];
```

Comment:
Similarly to the previous comment, consider renaming `separadorTo` to `toSeparator` for consistency and clarity.